api/views/user_views.py

```diff
@@ -414,6 +419,7 @@ def post(self, request, *args, **kwargs):
 
         except Exception as e:
             transaction.set_rollback(True)
+            logger.critical(f"Failed to upload audio: {str(e)}")
```

Comment:
We should have a document explaining the log levels and when to use which; why do you this is critical and "Failed to delete user files.." is an errror?

Reply:
In retrospect 'error' fits both better.